locator/everywhere_replication_strategy.cc

```diff
@@ -41,6 +41,11 @@ void everywhere_replication_strategy::validate_options(const gms::feature_servic
 }
 
 sstring everywhere_replication_strategy::sanity_check_read_replicas(const effective_replication_map& erm, const inet_address_vector_replica_set& read_replicas) const {
+    const auto& replication_strategy = erm.get_replication_strategy();
+    if (replication_strategy.uses_tablets()) {
```

Comment:
We're inside `everywhere_replication_strategy` code, so `erm.get_replication_strategy()` is this strategy, no? In which case this branch is dead code (only NTS can use tablets)